Class fields are now created to be lower camel case. Added interation test.

MungoRae opened the pull request:
Fix for issue #563. Made fields lower camel case.

Thanks

samskiter commented:
Looks to me. i'm surprised you had to make a new method in nameutils... so I'm not sure that there's a test for schema either. I just had a look but couldn't find anything. Could you perhaps add a test for that input type too? I would probably expect it to be in `PropertyIT`

joelittlejohn commented:
Yes, agreed. @MungoRae could you move your integration test to PropertiesIT? We can merge this for the next release.

MungoRae commented on jsonschema2pojo-integration-tests/src/test/resources/schema/ref/httpRefs.json:
```diff
@@ -2,10 +2,10 @@
     "type" : "object",
     "properties" : {
         "address" : {
-            "$ref" : "https://jsonschema2pojo.googlecode.com/git-history/jsonschema2pojo-0.3.4/jsonschema2pojo-integration-tests/src/test/resources/schema/ref/address.json"
```
This was failing one of the tests so I replaced it with a working URL to make sure the test passed. Not sure if this is the URL that should be used. Oddly enough this seemed to be passing yesterday.

MungoRae commented:
I updated the integration test to be inside of PropertiesIT and I changed it to use a schema instead to fit in with the rest of the tests in that set.

joelittlejohn commented:
Great, thanks! I guess some last piece of Google Code was finally killed off.
